src/Components/Aspire.Keycloak/AspireKeycloakExtensions.cs

```diff
@@ -0,0 +1,103 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.Hosting;
```

Comment:
```suggestion
namespace Microsoft.Extensions.DependencyInjection;
```

We put most if not all of our AuthenticationBuilder extension method in the M.E.DI namespace.

Reply:
Done!